docs/stack.md

```diff
@@ -30,7 +30,7 @@ We use [wagtail-modeltranslations](https://github.com/infoportugal/wagtail-model
 
 #### A/B testing
 
-We use [wagtail-experiments](https://github.com/torchbox/wagtail-experiments) for CMS-based A/B testing. Please see its documentation for more information.
+We currently don't have any A/B testing going on. We used to use [wagtail-experiments](https://github.com/torchbox/wagtail-experiments) but we're going to switch over to [wagtail-ab-testing](https://github.com/torchbox/wagtail-ab-testing)
```

Comment:
We don't think wagtail-experiments are being used. It's an old app that doesn't support Django 3 so I removed it. We'll be adding in wagtail-ab-testing in a coming sprint at some point as well, which would have removed this dependency anyway